library/VTKExtensions/Rendering/vtkF3DInteractorStyle.cxx

```diff
@@ -114,7 +114,19 @@ void vtkF3DInteractorStyle::Dolly()
   {
     return;
   }
-  this->Superclass::Dolly();
+
```

Comment:
`assert(this->CurrentRenderer)` to be safe ?

Reply:
I'm ok with that if you prefer that over the `if(this->CurrentRenderer)` that was there previously